sdk/automanage/Microsoft.Azure.Management.AutoManage/tests/Microsoft.Azure.Management.AutoManage.Tests.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.test.reference.props" />
+  <PropertyGroup>
+    <PackageId>AutoManage.Tests</PackageId>
+    <Description>AutoManage.Tests Class Library</Description>
+    <AssemblyName>AutoManage.Tests</AssemblyName>
+    <VersionPrefix>0.1.0-preview</VersionPrefix>
+    <RootNamespace>Microsoft.Azure.Management.AutoManage.Tests</RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Update="SessionRecords\OperationsTests\ListOperationsTest.json">
```

Comment:
Test failure caused by not finding SessionRecords. Please replace with following to copy over the **all** of the files
`  <ItemGroup>
    <None Update="SessionRecords\**\*.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </None>
  </ItemGroup>`

Reply:
thank you Allen, fixed.